action.yml

```diff
@@ -13,21 +13,37 @@ inputs:
   text:
     description: "The message text to post"
     required: true
+  unfurl_links:
+    description: "Whether links in the message should be unfurled"
+    type: boolean
+    default: true
 outputs:
   ts:
     description: "The timestamp ID of the message that was just posted"
+    value: ${{steps.slack-post-message.outputs.ts}}
 branding:
   icon: 'tag'
   color: 'blue'
 runs:
   using: 'composite'
   steps:
-    - run: |
-        {
-          echo "ts=<<EOF"
-          curl --silent --show-error -X POST -H "Authorization: Bearer ${{ env.SLACK_BOT_TOKEN || inputs.token }}" -H "Content-Type: application/json; charset=utf-8" --url https://slack.com/api/chat.postMessage \
-            -d "{\"channel\": \"${{ inputs.channel }}\", \"thread_ts\":\"${{ inputs.thread_ts }}\", \"text\":\"${{ inputs.text }}\"}" \
-            | jq --raw-output '.ts'
-          echo "EOF"
-        } >> "$GITHUB_OUTPUT"
+    - name: Validate required inputs
       shell: bash
+      run: |
+        [[ "${{ inputs.channel }}" ]] || { echo "required input 'channel' not specified" ; exit 1; }
+        [[ "${{ inputs.text }}" ]] || { echo "required input 'text' not specified" ; exit 1; }
```

Comment:
Not strictly what I set out to cover here, but it felt reasonable to validate the required inputs explicitly to (hopefully) provide a better troubleshooting experience.

Reply:
👍